config/rbac/role.yaml

```diff
@@ -101,6 +101,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
@Jooho Just FYI, since I think this is related to previous changes from you.

Reply:
@israel-hdez yea but I think I didn't use the one so I will take it out later. Please include it now